cli_tools/google-osconfig-agent/ospatch/patch_run.go

```diff
@@ -356,50 +340,66 @@ func ackPatch(ctx context.Context, patchJobName string) {
 	// the server may have inadvertantly notified us twice (at least once deliver) so we
 	// can ignore it.
 	if currentPatchJob == nil || currentPatchJob.Job.PatchJobName != patchJobName {
-		res, err := reportPatchDetails(ctx, patchJobName, osconfigpb.Instance_NOTIFIED, 0, "")
+		client, err := osconfig.NewClient(ctx, option.WithEndpoint(config.SvcEndpoint()), option.WithCredentialsFile(config.OAuthPath()))
 		if err != nil {
+			logger.Errorf("osconfig.NewClient Error: %v", err)
+			return
+		}
+
+		pr := &patchRun{ctx: ctx, client: client, Job: &patchJob{&osconfigpb.ReportPatchJobInstanceDetailsResponse{PatchJobName: patchJobName}}}
+		if err := pr.reportPatchDetails(osconfigpb.Instance_NOTIFIED, 0, ""); err != nil {
```

Comment:
Shouldn't this now be `ACKED` or is that in a future PR?

Reply:
Thats in a future PR, after i get the proto synced